packages/nuxt/src/module.ts

```diff
@@ -12,7 +12,7 @@ export default defineNuxtModule<ModuleOptions>({
     name: 'FormKit',
     configKey: 'formkit',
     compatibility: {
-      nuxt: '^3.0.0',
+      nuxt: '^3.0.0-rc.1',
```

Comment:
As this module has the edge version of Nuxt Kit, I'd suggest requiring the latest edge version of Nuxt (consulted with Nuxt's framework lead):
```suggestion
      nuxt: '^3.0.0-rc.10',
```

Reply:
> As this module has the edge version of Nuxt Kit, I'd suggest requiring the latest RC version of Nuxt (consulted with Nuxt's framework lead):

I think you're right, we should move away from edge now that things are getting stable. However, wouldn't the appropriate change be in the package.json, not the module? I'm not sure we want to drop support for `rc.1-rc.9`

```json
{
  "dependencies": {
  "@formkit/vue": "1.0.0-beta.10",
  "@nuxt/kit": "npm:@nuxt/kit-edge@latest"
}
```

Reply:
I'll take your word for it @pi0! We'll get that updated in our beta.11